app/views/overrides/organizations/_edit_override.html.erb

```diff
@@ -8,11 +8,19 @@
     <%= selectable_f(f, :service_level, options_for_select(service_level_options, service_level_selected), {}, :label => _('Default System SLA')) %>
   <% end %>
 
+
   <%= field(f, _('Debug Certificate'),
             :help_inline => _('This certificate allows a user to view the repositories in any environment from a browser.')) do
     button_tag _('Generate and Download'), :id => :download_debug_cert_key,
                :type => :button, :style => 'margin-top: 8px;', :class => 'btn btn-small',
                'data-url' => "/katello/api/v2/organizations/#{@taxonomy.id}/download_debug_certificate"
   end %>
 
-<% end%>
+  <%= field(f, _('Simple Content Access'),
+            :help_inline => _('Toggling Simple Content Access will refresh your manifest.')) do
+    tag1 = hidden_field_tag 'simple_content_access', '0'
```

Comment:
Yeah, Rails does it for you if you use the form helpers.  (see ["Gotcha"](https://api.rubyonrails.org/v6.0.3.2/classes/ActionView/Helpers/FormHelper.html#method-i-check_box)) But since it's not an attribute of `Organization` that we're editing I just decided to do it myself :)